Makefile

```diff
@@ -63,7 +63,16 @@ test: lint pythoncheck
 	pytest -n 8 -x -m convert_maxpool
 
 
-lint: pythoncheck
+CONVERT_DIR=$(CURRENT_DIR)/tf_encrypted/convert
```

Comment:
this can probably be done more cleanly, but I'm new to make so would appreciate a pointer

Reply:
You run make from the project root so you shouldn't need the `$(CURRENT_DIR)` :) Otherwise, looks great!